lib/utils/device/deploy.ts

```diff
@@ -46,7 +46,7 @@ export interface DeviceDeployOptions {
 	dockerfilePath?: string;
 	registrySecrets: RegistrySecrets;
 	nocache: boolean;
-	live: boolean;
+	nolive: boolean;
```

Comment:
Minor: it would nice to accept and ignore a `--live` flag for users that have already been using it...

Reply:
`--live` is already ignored by capitano

Reply:
ha, good point! Capitano could use this example to claim that ignoring flags is a feature rather than a bug. :-D

Reply:
Let's not go there... :P